### PR TITLE
docs(cockpit): add set variables on migration documentation

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-13584

### CODE REVIEW

File: content/webapps/cockpit/bpmn/process-instance-migration.md [hunk 1]
```diff
@@ -80,12 +80,32 @@ You can visualize the migration plan in two ways: You can hover over an activity
...
+After finishing set variables, click the `Select Instances` button to proceed to instance selection. Initially, all instances of the source process definition version are displayed. You can use the filter bar above the list of instances to filter the displayed instances. Furthermore, you can copy a link to the current filter query to your clipboard by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-link"></i></button> button and you can save filter queries to your local browser storage by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-floppy-disk"></i></button> button and inserting a name in the drop down menu that appears. You can then retrieve the filter query by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-floppy-disk"></i></button> button and selecting the chosen name in the drop down menu.
```
Comment (marstamm) [inline content/webapps/cockpit/bpmn/process-instance-migration.md:108]: The grammar seems a bit odd to me. I think it should be "After finishing setting variables, ...". If we reference the Tab here, I think it should be `**Set Variables**` .

**Review by marstamm — COMMENTED** [review]
LGTM, one comment below for consideration :slightly_smiling_face:

File: content/webapps/cockpit/bpmn/process-instance-migration.md (hunk 1 above)
Comment (tasso94) [reply content/webapps/cockpit/bpmn/process-instance-migration.md:108]:
```suggestion
After finishing the **Set Variables** step, click the `Select Instances` button to proceed to instance selection. Initially, all instances of the source process definition version are displayed. You can use the filter bar above the list of instances to filter the displayed instances. Furthermore, you can copy a link to the current filter query to your clipboard by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-link"></i></button> button and you can save filter queries to your local browser storage by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-floppy-disk"></i></button> button and inserting a name in the drop down menu that appears. You can then retrieve the filter query by clicking on the <button class="btn btn-xs"><i class="glyphicon glyphicon-floppy-disk"></i></button> button and selecting the chosen name in the drop down menu.
```

Comment (tasso94) [reply content/webapps/cockpit/bpmn/process-instance-migration.md:108]: 👍

**Pull request merged**